### PR TITLE
fix: skip camelCase transform for pnl debug json download

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/services/axios-tranformers.ts [hunk 1]
```diff
@@ -61,11 +61,13 @@ const jsonTransformer: AxiosResponseTransformer = (data, headers) => {
...
-    jsonTransformer,
-    skipRoot ? noRootCamelCaseTransformer : camelCaseTransformer,
-  ];
+export function setupTransformer(camelCaseTransform = true, skipRoot = false): AxiosResponseTransformer[] {
+  const transformer = [jsonTransformer];
+
+  if (camelCaseTransform)
+    transformer.push(skipRoot ? noRootCamelCaseTransformer : camelCaseTransformer);
+
+  return transformer;
```
Comment (kelsos) [inline frontend/app/src/services/axios-tranformers.ts:70]: do we really need to do this change?

File: frontend/app/src/composables/api/task.ts [hunk 2]
```diff
@@ -27,9 +28,10 @@ export function useTaskApi() {
     return handleResponse(response);
   };
 
-  const queryTaskResult = async <T>(id: number): Promise<ActionResult<T>> => {
+  const queryTaskResult = async <T>(id: number, transformer?: AxiosResponseTransformer[]): Promise<ActionResult<T>> => {
     const config: Partial<AxiosRequestConfig> = {
       validateStatus: validTaskStatus,
+      transformResponse: transformer ?? basicAxiosTransformer,
```
Comment (kelsos) [inline frontend/app/src/composables/api/task.ts:34]: no need to pass the basicTransformer here, either override with the custom one or let the defaults

File: frontend/app/src/services/axios-tranformers.ts (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/src/services/axios-tranformers.ts:70]: either this, or I need to import jsonTransformer directly in the awaitTask call for the pnl debug download

File: frontend/app/src/store/reports/index.ts [hunk 3]
```diff
@@ -312,7 +313,7 @@ export const useReportsStore = defineStore('reports', () => {
         TaskType.TRADE_HISTORY,
         {
           title: t('actions.reports.generate.task.title'),
-          // TODO skip transform?
+          transformer: setupTransformer(false),
```
Comment (kelsos) [inline frontend/app/src/store/reports/index.ts:316]: Can't we just pass the jsonTransformer directly in this case?

**Review by kelsos — APPROVED** [review]
lgtm thanks :)

**Pull request merged**